include/flatbuffers/code_generators.h

```diff
@@ -129,6 +129,8 @@ struct CommentConfig {
   const char *last_line;
 };
 
+static const std::string GeneratedWarning("automatically generated by the FlatBuffers compiler,"
```

Comment:
Why was this moved here? Seems an entirely independent change from what you're trying to accomplish, and actually makes things worse.